test/system/160-volumes.bats

```diff
@@ -558,4 +558,36 @@ EOF
 }
 
 
+@test "podman volume globs" {
+    vol1a=${PODMAN_TMPDIR}/v1_$(random_string)
+    vol1b=${PODMAN_TMPDIR}/v1_$(random_string)
+    vol2=${PODMAN_TMPDIR}/v2_$(random_string)
+    touch $vol1a $vol1b $vol2
+
+    # if volumes source and dest match then pass
+    run_podman run --rm -v ${PODMAN_TMPDIR}/v1\*:ro $IMAGE ls $vol1a $vol1b
```

Comment:
What happens when I specify a source glob _and_ a destination?

Reply:
It will fail.